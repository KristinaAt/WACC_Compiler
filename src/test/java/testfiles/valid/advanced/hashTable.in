a
5
a
10
b
5
b
6
c
d
e
10
e
10
c
d
f
c
d
g