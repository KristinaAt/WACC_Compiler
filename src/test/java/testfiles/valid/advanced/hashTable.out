===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: Please enter an integer to insert: You have entered: 5
Successfully insert it. The integer is new.
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: Please enter an integer to insert: You have entered: 10
Successfully insert it. The integer is new.
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: Please enter an integer to find: You have entered: 5
Find the integer.
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: Please enter an integer to find: You have entered: 6
The integer is not found.
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: There are 2 integers.
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: Here are the integers:510
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: Please enter an integer to remove: You have entered: 10
The integer has been removed.
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: Please enter an integer to remove: You have entered: 10
The integer is not found.
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: There is only 1 integer.
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: Here are the integers:5
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: All integers have been removed.
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: There are 0 integers.
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: Here are the integers:
===========================================
========== Hash Table Program =============
===========================================
=                                         =
= Please choose the following options:    =
=                                         =
= a: insert an integer                    =
= b: find an integer                      =
= c: count the integers                   =
= d: print all integers                   =
= e: remove an integer                    =
= f: remove all integers                  =
= g: exit                                 =
=                                         =
===========================================
Your decision: Goodbye Human
